<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomKiosk</class>
 <widget class="QMainWindow" name="RoomKiosk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>회의실 키오스크</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0">
    <item>
     <widget class="QLabel" name="label_title">
      <property name="text">
       <string>인증번호를 입력하세요</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_code">
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="placeholderText">
       <string>인증번호 4자리</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_enter">
        <property name="text">
         <string>입실</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_status">
        <property name="text">
         <string>상태조회</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lbl_msg">
      <property name="text">
       <string>대기 중…</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_status">
      <property name="text">
       <string>HVAC 연결 상태 표시</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
